mmhuman3d/data/data_converters/gta_human.py

```diff
@@ -0,0 +1,293 @@
+            model_path='data/body_models/smpl',
+            extra_joints_regressor='data/body_models/J_regressor_extra.npy')
+    ).to(device)
+
+    def convert(self, dataset_path: str, out_path: str) -> dict:
+        """
+        Args:
+            dataset_path (str): Path to directory where raw images and
+            annotations are stored.
+            out_path (str): Path to directory to save preprocessed npz file
+
+        Returns:
+            dict:
+                A dict containing keys video_path, smplh, meta, frame_idx
+                stored in HumanData() format
+        """
+        # use HumanData to store all data
+        human_data = HumanData()
+
+        body_pose_list = []
+        global_orient_list = []
+        betas_list = []
+        transl_list = []
+
+        image_path_list = []
+        bbox_xywh_list = []
+
+        keypoints_2d_gta_list = []
+        keypoints_3d_gta_list = []
+        keypoints_2d_list = []
+        keypoints_3d_list = []
+
+        anno_paths = sorted(
+            glob.glob(os.path.join(dataset_path, 'annotations', '*.pkl')))
+
+        for anno_path in tqdm(anno_paths):
+
+            with open(anno_path, 'rb') as f:
+                anno = pickle.load(f, encoding='latin1')
+
+            base = os.path.basename(anno_path)  # -> seq_00000001.pkl
+            seq_idx, _ = os.path.splitext(base)  # -> seq_00000001
+            num_frames = len(anno['body_pose'])
+
+            # keypoints_2d_gta_openpose = map_joints(
+            # annotation['keypoints_2d'],
+            # joint_mapper_gta_to_openpose,
+            # (num_frames, 25, 3))  # 25 joints, (x, y, conf)
+            # keypoints_2d_gta_joint24 = map_joints(annotation['keypoints_2d'],
+            # joint_mapper_gta_to_joint24,
+            # (num_frames, 24, 3))  # 24 joints, (x, y, conf)
+            # keypoints_3d_gta_joint24 = map_joints(annotation['keypoints_3d'],
+            # joint_mapper_gta_to_joint24,
+            # (num_frames, 24, 4))  # 24 joints, (x, y, z, conf)
+
+            keypoints_2d_gta, keypoints_2d_gta_mask = convert_kps(
+                anno['keypoints_2d'], src='gta', dst='smpl_49')
+            keypoints_3d_gta, keypoints_3d_gta_mask = convert_kps(
+                anno['keypoints_3d'], src='gta', dst='smpl_49')
+
+            global_orient = anno['global_orient']
+            body_pose = anno['body_pose']
+            betas = anno['betas']
+            transl = anno['transl']
+
+            # keypoints_2d_smpl_openpose, \
+            # keypoints_2d_smpl_joint24, \
+            # keypoints_3d_smpl_openpose, \
+            # keypoints_3d_smpl_joint24 = \
+            #     extract_keypoints_from_smpl(smpl,
+            #     global_orient=torch.tensor(global_orient, device=device),
+            #     body_pose=torch.tensor(body_pose, device=device),
+            #     betas=torch.tensor(betas, device=device),
+            #     transl=torch.tensor(transl, device=device)
+            #     )
```

Comment:
Do you plan to leave these comments in the script? If you do, you might want to consider adding some comments to indicate what it's used for

Reply:
Additionally, please add the dataset citation and recommend file structure for gta-human to `preprocess_datasets.md` and `keypoints_convention.md`

Reply:
> Do you plan to leave these comments in the script? If you do, you might want to consider adding some comments to indicate what it's used for

All commented out lines are removed.

Reply:
> Additionally, please add the dataset citation and recommend file structure for gta-human to `preprocess_datasets.md` and `keypoints_convention.md`

Added.